itou/common_apps/organizations/models.py

```diff
@@ -65,10 +66,14 @@ def has_member(self, user):
     def has_admin(self, user):
         return self.active_admin_members.filter(pk=user.pk).exists()
 
-    def add_member(self, user):
-        is_admin = self.members.count() == 0
-        membership = self.memberships.create(user=user, is_admin=is_admin)
-        membership.save()
+    def add_or_activate_member(self, user):
+        """Add user to organization members, or activate membership if already there"""
+
+        if not self.memberships.filter(user=user).update(is_active=True):
+            # There was no membership with this user
+            is_admin = self.members.count() == 0
+            membership = self.memberships.create(user=user, is_admin=is_admin)
+            membership.save()
```

Comment:
Tous les tests passent avec la modif suivante :
```suggestion
            self.memberships.create(user=user, is_admin=is_admin)
```